aws/scripts/py/cache_invalidation.py

```
@@ -0,0 +1,97 @@
+import json
+import subprocess
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)